src/ui/Header/Header.jsx

```diff
@@ -52,12 +52,14 @@ const TopDropdownMenu = ({
       <Dropdown.Menu role="group">{children}</Dropdown.Menu>
     </Dropdown>
   );
-  return (
-    <>
-      <Component />
-      {mobileText && <Component mobileText={mobileText} />}
-    </>
-  );
+  if (typeof window !== 'undefined') {
+    const resolution = window?.innerWidth;
+    const isMobile = resolution >= 320 && resolution <= 480;
```

Comment:
@nileshgulia1 

Remove `resolution >= 320` check, it should be mobile also under 320px

And also I think the breaking point is at `480` thus resolution should be strictly  `resolution < 480` and start behaving mobile at `479px`